### PR TITLE
document custom initiator iqn

### PR DESCRIPTION
**Pull request by rootfs**
cc @childsb 

@ahardin-rh can this be cherry-picked into 3.9 and 3.10? thanks!

### CODE REVIEW

File: install_config/persistent_storage/persistent_storage_iscsi.adoc [hunk 1]
```diff
@@ -122,3 +122,35 @@ spec:
...
+
+[[iscsi-custom-iqn]]
+
+=== iSCSI Custom Initiator IQN
+You can configure custom initiator IQN. This is helpful if the iSCSI targets are restricted to certain IQNs but the nodes that the iSCSI PVs are attached 
+are not guaranteed to have these IQNs.
+
+To specify custom initiator IQN, use `initiatorName` field. For example:
+
+====
```
Comment (ahardin-rh) [inline install_config/persistent_storage/persistent_storage_iscsi.adoc]: To specify the custom initiator IQN, use the `initiatorName` field.

File: install_config/persistent_storage/persistent_storage_iscsi.adoc [hunk 2]
```diff
@@ -122,3 +122,35 @@ spec:
...
+  iscsi:
+    targetPortal: 10.0.0.1:3260
+    portals: ['10.0.2.16:3260', '10.0.2.17:3260', '10.0.2.18:3260'] 
+    iqn: iqn.2016-04.test.com:storage.target00
+    lun: 0
+    initiatorName: iqn.2016-04.test.com:custom.iqn <1>
+    fsType: ext4
+    readOnly: false
+----
+<1> Add additional custom initiator IQN, use `initiatorName` field.
```
Comment (ahardin-rh) [inline install_config/persistent_storage/persistent_storage_iscsi.adoc]: To add an additional custom initiator IQN, use the `initiatorName` field.

**Comment by ahardin-rh** [comment]
@qinpingli Can you please give a QE review?

@tmorriso-rh FYI

**Comment by qinpingli** [comment]
@ahardin-rh I am ok with you comments.

**Comment by ahardin-rh** [comment]
@rootfs Once you make the suggested updates, we can get this merged. Thanks!

**Comment by rootfs** [comment]
@ahardin-rh updated, thanks!

**Pull request merged**